tests/test_types_payment_card_number.py

```diff
@@ -9,7 +9,9 @@
 
 VALID_AMEX = '370000000000002'
 VALID_MC = '5100000000000003'
-VALID_VISA = '4050000000000001'
+VALID_VISA_16 = '4050000000000001'
+VALID_VISA_13 = '4050000000001'
+VALID_VISA_19 = '4050000000001'
```

Comment:
This should probably be 19 characters, and sorted by length...
```suggestion
VALID_VISA_13 = '4050000000001'
VALID_VISA_16 = '4050000000000001'
VALID_VISA_19 = '4050000000000000001'
```

Reply:
Yes, thank you. Corrected